extensions/haskell/extension.toml

```diff
@@ -4,7 +4,7 @@ description = "Haskell support."
 version = "0.1.0"
 schema_version = 1
 authors = [
-    "Pseudomata <pseudomata@proton.me>",
+    "Pocæus <github@pocaeus.com>",
```

Comment:
can s/`æ`/`ae` if special characters cause issues.

Reply:
Looks like it should render fine:

<img width="679" alt="Screenshot 2024-06-19 at 10 28 06 AM" src="https://github.com/zed-industries/zed/assets/1486634/35ab2898-a705-42d4-abaf-8325ef6750fb">